packages/models-library/src/models_library/api_schemas_webserver/storage.py

```diff
@@ -42,3 +44,22 @@ class BatchDeletePathsBodyParams(InputSchema):
 
 class DataExportPost(InputSchema):
     paths: list[PathToExport]
+
+
+def validate_pattern_has_enough_characters(v: str) -> str:
```

Comment:
this validator can be handy for other search that includes globs. I would put it in  packages/models-library/src/models_library/utils/common_validators.py